### PR TITLE
Login/Sign up page

### PR DESCRIPTION
**Pull request by rzayevan**
Built off Kim's PR:
https://github.com/rzayevan/pokerpals-group13/pull/14

### CODE REVIEW

File: poker-pals/src/components/Login.vue [hunk 1]
```diff
@@ -1,14 +1,174 @@
...
+      <div class="col-md-6">
+        <!-- Login Form -->
+        <form class="text-left" v-on:submit="loginAction" v-if="showLogin">
+          <div class="form-group">
+            <label for="email">Email</label>
+            <input class="form-control" type="email" id="loginEmail" v-model="loginData.email" required/>
+          </div>
+
+          <div class="form-group">
+            <label for="password1">Password</label>
```
Comment (aidan-lee) [inline poker-pals/src/components/Login.vue]: What is password1? Is it the first of two fields where you need to enter your new password when you're signing up for an account?  I'm not sure if the for field gets used again anywhere but is it worth making password1 and password2 more descriptive?

Comment (rzayevan) [reply poker-pals/src/components/Login.vue]: I'll update the ids

**Review by aidan-lee — APPROVED** [review]

File: server/utilities/UserUtils.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    });
+
+    return matchFound;
+}
+
+/**
+ * Checks whether or not the provided email is unique
+ */
+exports.emailExists = function(user) {
+    let users = DataAccessLayer.ReadUsersFile();
```
Comment (darbrius) [inline server/utilities/UserUtils.js]: Use the cache, don't read from the file

File: server/utilities/UserUtils.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+const DataAccessLayer = require('../controllers/DataAccessLayer.js')
+
+/**
+ * Checks if the provided credentials match the ones stored in the application
+ */
+exports.credentialsMatch = function(user) {
+    let users = DataAccessLayer.ReadUsersFile();
```
Comment (darbrius) [inline server/utilities/UserUtils.js]: Use the cache, don't read from the file

**Pull request merged**